### DISCUSSION TITLE
autocomplete for #include never shows .c&.cpp files. only shows headers .h files

### DISCUSSION THREAD

**Discussion by masriomarm** [microsoft/vscode-cpptools#8894] [Q&A]
Greetings,
i have an issue as described in the title.
when trying to include a .c/.cpp file the autocomplete never suggests  source files. it shows many file types but never source files. it shows .h files, .txt files, .pdf files, and many others. but never list my .c/.cpp files.
i tried to search in setting but couldn't find any good results.
any idea how to solve that?

**Comment by sean-mcmanus** [comment]
This is a duplicate of https://github.com/microsoft/vscode-cpptools/discussions/8891
